### PR TITLE
Cmor 3.2.7

### PR DESCRIPTION
**Pull request by dnadeau4**

### CODE REVIEW

File: recipes/cmor/meta.yaml.in [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - gcc [osx]
+  imports:
+    - cmor
+
+requirements:
+  build:
+    - python
+    - ossuuid 1.6.2
+    - udunits2
+    - hdf5 1.8.17
```
Comment (doutriaux1) [inline recipes/cmor/meta.yaml.in]: @dnadeau4 hdf5 1.8.18 please

Comment (doutriaux1) [reply recipes/cmor/meta.yaml.in]: @dnadeau4 you can get recipe from http://github.com/uv-cdat/conda-recipes

File: recipes/cmor/meta.yaml.in [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - libnetcdf 4.4.1
```
Comment (doutriaux1) [inline recipes/cmor/meta.yaml.in]: 4.4.*

**Comment by doutriaux1** [comment]
cmor.yml should be updated as well.

File: recipes/cmor/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    - ossuuid 1.6.2
+    - udunits2
+    - libnetcdf 4.4.*
+    - numpy x.x
+    - cdms2
+    - libgcc [osx]
+    - hdf5 1.8.18
+
+about:
+    home:  http://gitub.com/PCMDI
```
Comment (durack1) [inline recipes/cmor/meta.yaml]: @dnadeau4 @doutriaux1 the `gitub.com` needs to be updated to `gitHub.com` (note the missing 'h')

File: recipes/cmor/meta.yaml.in [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    - ossuuid 1.6.2
+    - udunits2
+    - libnetcdf 4.4.1
+    - numpy x.x
+    - cdms2
+    - gcc [osx]
+    - hdf5 1.8.17
+
+about:
+    home:  http://gitub.com/PCMDI
```
Comment (durack1) [inline recipes/cmor/meta.yaml.in]: @dnadeau4 @doutriaux1 the `gitub.com` needs to be updated to `github.com` (note the missing 'h')

File: recipes/cmor/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+package:
+    name: cmor
+    version: 3.2.7
+
+source:
+    git_rev: master
+    git_url: git://github.com/PCMDI/cmor.git
```
Comment (durack1) [inline recipes/cmor/meta.yaml]: I also wonder about the `git://` vs `https://` which is what is noted in the documentation https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#source-from-git

@dnadeau4 @doutriaux1

File: recipes/cmor/meta.yaml.in [hunk 6]
```diff
@@ -0,0 +1,40 @@
+package:
+    name: cmor
+    version: @VERSION@
+
+source:
+    git_rev: @UVCDAT_BRANCH@
+    git_url: git://github.com/PCMDI/cmor.git
```
Comment (durack1) [inline recipes/cmor/meta.yaml.in]: I also wonder about the `git://` vs `https://` which is what is noted in the documentation https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#source-from-git

@dnadeau4 @doutriaux1

File: recipes/cmor/meta.yaml (hunk 5 above)
Comment (doutriaux1) [reply recipes/cmor/meta.yaml]: actually it probably ought to be: https://pcmdi.github.io/cmor-site/

Comment (durack1) [reply recipes/cmor/meta.yaml]: @doutriaux1 you're getting the `source` vs `about` entries confused.

`source` should be `https://` and `about` should be https://cmor.llnl.gov/

File: recipes/cmor/meta.yaml.in (hunk 1 above)
Comment (dnadeau4) [reply recipes/cmor/meta.yaml.in]: Done in master

File: recipes/cmor/meta.yaml.in (hunk 2 above)
Comment (dnadeau4) [reply recipes/cmor/meta.yaml.in]: Done in master

File: recipes/cmor/meta.yaml.in (hunk 4 above)
Comment (dnadeau4) [reply recipes/cmor/meta.yaml.in]: will do.

File: recipes/cmor/meta.yaml (hunk 5 above)
Comment (doutriaux1) [reply recipes/cmor/meta.yaml]: @durack1 yes typed reply in wrong box. I meant for the "home" entry

File: recipes/cmor/meta.yaml.in (hunk 4 above)
Comment (dnadeau4) [reply recipes/cmor/meta.yaml.in]: Done

File: recipes/cmor/meta.yaml.in (hunk 6 above)
Comment (dnadeau4) [reply recipes/cmor/meta.yaml.in]: Conda does not mind it and build works.

**Pull request merged**